lib/active_merchant/billing/gateways/adyen.rb

```diff
@@ -157,7 +157,9 @@ def scrub(transcript)
           gsub(%r((Authorization: Basic )\w+), '\1[FILTERED]').
           gsub(%r(("number\\?":\\?")[^"]*)i, '\1[FILTERED]').
           gsub(%r(("cvc\\?":\\?")[^"]*)i, '\1[FILTERED]').
-          gsub(%r(("cavv\\?":\\?")[^"]*)i, '\1[FILTERED]')
+          gsub(%r(("cavv\\?":\\?")[^"]*)i, '\1[FILTERED]').
```

Comment:
I know you followed the existing pattern, but this is a good opportunity to tighten up these RegExp's even more.  

Currently these do NOT get filtered :/ (notice the space between the colons).

'"cvc" : "addafd", "bankLocationId" : "aaa"'

A potential way to prevent this is to surround the `:` with `\s*` -> `gsub(%r(("cvc\\?"\s*:\s*\\?")[^"]*)i, '\1[FILTERED]')` etc.